### DISCUSSION TITLE
Help me, I have a hacker

### DISCUSSION THREAD

**Discussion by Janeczek027** [community/community#121082] [Other Feature Feedback, Questions, & Ideas]
For some time now I have had a pesky hacker, he blocks my websites, changes my phone settings, installs applications, etc., I have no idea how to fight him, please help

**Comment by Amewu22** [comment]
Sorry for the inconvenient, please provide your Pesky ID number to slove your problem.

**Comment by Akash1134** [comment]
Hi @Janeczek027 , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
